README.md

```diff
@@ -266,9 +266,13 @@ wss.on('connection', function connection(ws, request, client) {
 });
 
 server.on('upgrade', function upgrade(request, socket, head) {
+  /**
+   * NOTE:
+   *  The `authenticate` function not implemented! Please replace that by your auth logic!
```

Comment:
I don't think this is needed. If the example is copy-pasted an error will be thrown as the function is not defined.

Reply:
Sure,  I thought about it because of https://github.com/websockets/ws/issues/1674 and https://github.com/websockets/ws/issues/1629 (maybe there are similar issues).